source/manual/replicate-app-data-locally.html.md

```diff
@@ -51,6 +51,13 @@ dump. Then, from `govuk-puppet/development-vm/replication` run:
 
 See [running out of disk space in development](/manual/development-disk-space.html).
 
+### `govuk data` tool
+If you are not able to free up enough space to replicate all data, you may consider using the `govuk data` tool to list and load particular data, downloaded through the replication scripts. For more information see the [`govuk data` tool
+docs](https://github.com/alphagov/govuk-guix/blob/master/doc/local-data.md).
+
+You may need to set up your CLI access for AWS in the VM by creating `~/.aws/config` and `~/.aws/credentials` in the VM.
```

Comment:
Should we clarify this bit saying that we can copy these two from our local machine?

Reply:
Yes, definitely. My first version had this info. 
Does this sound ok:
You may need to set up your CLI access for AWS in the VM by copying `~/.aws/config` and `~/.aws/credentials` from your host machine into the VM.